app/src/atoms/StatusLabel/index.tsx

```diff
@@ -15,18 +17,28 @@ interface StatusLabelProps {
   backgroundColor: string
   iconColor: string
   textColor?: string
+  fontWeight?: number
```

Comment:
```suggestion
  fontWeight?: number
  size?: number
```
https://github.com/Opentrons/opentrons/pull/11446#issuecomment-1242081860